src/devices/src/virtio/net/device.rs

```diff
@@ -459,10 +471,13 @@ impl Net {
             FrontendError::AddUsed
         })?;
 
-        self.rx_deferred_irqs = true;
         METRICS.net.rx_bytes_count.add(frame_len);
         METRICS.net.rx_packets_count.inc();
 
+        self.num_deffered_irqs += used_buffs.len() as u16;
```

Comment:
nit: rename `num_deffered_irqs` to smth like `used_bytes_pending_irq` or similar.